scripts/hab2_benchmark.py

```diff
@@ -0,0 +1,238 @@
+import argparse
```

Comment:
what/who is supposed to use this script? If it is the end user, this should be documented better.

Reply:
This is and `bench_runner.sh` run the Habitat 2.0 simulator benchmark. @aclegg3 can you add additional instructions for these two files?

Reply:
@aclegg3 How often is the bench_runner.sh used? It looks like it only uses the idle configurations. Do you think we could remove it?

Reply:
@vincentpierre and I talked about this offline.
My plan is to move all Hab2 benchmarking utils into a new subdirectory with a README and some documentation. I'll also add a test to run the benchmark code on CI.